### PR TITLE
{vis}[intel/2018b] Pillow-SIMD v5.3.0

### PR DESCRIPTION
**Pull request by dithwick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by dithwick** [comment]
Test report by @dithwick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz, Python 2.6.6
See https://gist.github.com/1c24a83364501e3fa4218ea9e05f707e for a full test report.

File: easybuild/easyconfigs/p/Pillow-SIMD/Pillow-SIMD-5.3.0-intel-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('libjpeg-turbo', '2.0.0'),
+    ('libpng', '1.6.34'),
+    ('zlib', '1.2.11'),
+    ('LibTIFF', '4.0.9'),
+    ('freetype', '2.9.1')
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/p/Pillow-SIMD/Pillow-SIMD-5.3.0-intel-2018b-Python-3.6.6.eb]: Please add:
```
use_pip = True
download_dep_fail = True
```

File: easybuild/easyconfigs/p/Pillow-SIMD/Pillow-SIMD-5.3.0-intel-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow-SIMD'
+version = '5.3.0'
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pillow-SIMD/Pillow-SIMD-5.3.0-intel-2018b-Python-3.6.6.eb]: Shouldn't the `post0` be reflected in the `version`?

**Comment by dithwick** [comment]
Test report by @dithwick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz, Python 2.6.6
See https://gist.github.com/6ca72eef60110300ad4bdfe4b7233461 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/f4f1977a74449300621af9a14b3870d5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2405.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7a71b1b407d41c464d7fd7ecbf3e1697 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2106.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/47b277ca46d1dc747b118c85b1a20032 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @dithwick!

**Pull request merged**